Client/Frontend/Widgets/ActionOverlayTableViewController.swift

```diff
@@ -155,10 +160,18 @@ class ActionOverlayTableViewHeader: UITableViewHeaderFooterView {
         let siteImageView = UIImageView()
         siteImageView.contentMode = UIViewContentMode.ScaleAspectFit
         siteImageView.clipsToBounds = true
-        siteImageView.layer.cornerRadius = SimpleHighlightCellUX.CornerRadius
         return siteImageView
     }()
 
+    lazy var siteImageBackground: UIView = {
```

Comment:
Is there a reason why you needed to make a new siteImageBackground? if you look at siteImageView in SimpleHighlightCell it works perfectly fine without having to use a separate view for a background color